vm/src/obj/objint.rs

```diff
@@ -90,6 +90,10 @@ impl FromPyObjectRef for BigInt {
     }
 }
 
+fn int_bool(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
+    arg_check!(vm, args, required = [(zelf, Some(vm.ctx.int_type()))]);
+    Ok(vm.ctx.new_bool(true))
```

Comment:
How about the `false` case? This never happens? It should be false when the int is zero right?

Reply:
yes, Sorry I missed it.